frontend/public/components/terminal.jsx

```diff
@@ -91,7 +106,11 @@ export class Terminal extends React.Component {
 
   render () {
     return <div ref={this.outerRef} style={this.state} className={this.props.className}>
-      <div ref={this.innerRef} className="console" />
+      <div ref={this.innerRef} className="console">
+        { this.isFullscreen && <button className="btn btn-link console-collapse-link" onClick={() => this.setFullscreen(false)}>
```

Comment:
onClick={this.setFullscreen(false)}

Reply:
If I do that the onClick executes immediately, think I need the fat arrow

Reply:
Ah you're right, sorry. You can remove the bind from the constructor in that case then

Reply:
ok, fixed, thanks